app/src/main/java/com/lastaoutdoor/lasta/utils/Constants.kt

```diff
@@ -0,0 +1,18 @@
+package com.lastaoutdoor.lasta.utils
+
+object Constants {
+  const val ACTION_SERVICE_START = "ACTION_SERVICE_START"
+  const val ACTION_SERVICE_STOP = "ACTION_SERVICE_STOP"
+  const val ACTION_SERVICE_CANCEL = "ACTION_SERVICE_CANCEL"
+
+  const val STOPWATCH_STATE = "STOPWATCH_STATE"
+
+  const val NOTIFICATION_CHANNEL_ID = "STOPWATCH_NOTIFICATION_ID"
+  const val NOTIFICATION_CHANNEL_NAME = "STOPWATCH_NOTIFICATION"
+  const val NOTIFICATION_ID = 10
+
+  const val CLICK_REQUEST_CODE = 100
+  const val CANCEL_REQUEST_CODE = 101
+  const val STOP_REQUEST_CODE = 102
+  const val RESUME_REQUEST_CODE = 103

```

Comment:
Well done ! A `Constants` class is ideal in this heavyly-abstract context

Reply:
Thank you, thought it would be useful in this case